abbr/views.py

```diff
@@ -31,7 +34,7 @@ def home_page():
         return jsonify("Hi, I'm abbr. "
                        "Usage: POST either a url as string or a dict with "
                        "'url' and optionally 'name' and 'expiry' "
-                       "('%Y-%m-%d %H:%M:%S').")
+                       "('%Y-%m-%d %H:%M:%S', Default: " + human(get_expiry(), precision=1) + ").")
```

Comment:
Since we return the format here, I think the message would look strange with a humanized value and be more consistent when using an ISO-ish date.